DOCS-7650-updated the content (#230)
mulesoft/docs-api-manager: modules/ROOT/pages/api-gateway-encryption-mule4.adoc

> anypoint.platform.client_secret.<n>

Reviewer: The <n> is *not* part of the key of the property. For example, client_id as declared in a generic way in the wrapper.conf file:
`wrapper.java.additional.<n>=-Danypoint.platform.client_id=XXXXXXXX` and the same, resolving the `<n>` placeholder to a unique value within the file, and using `123` as a correct client ID value.
`wrapper.java.additional.100=-Danypoint.platform.client_id=123`

Author: Done! Thanks!

anypoint.platform.proxy_password.<n>
--
Replace <n> with the unique number within the property `wrapper.java.additional` file.
[CAUTION]
--

> If you add `wrapper.java.additional.n` entries to the configuration file, you must change each instance of `<n>` to a consecutive number. otherwise, Java does not parse the properties correctly.

Reviewer: When using the generic way (which must be commented out in the file), use `wrapper.java.additional.<n>`.

Reviewer: If I remember correctly, having an uncommented `wrapper.java.additional.<n>` line somewhere in the file, will stop the runtime from starting.

Author: Done! Thanks!

You can encrypt your Anypoint Platform credentials (client ID, client secret, and your proxy password) and add those encrypted values, along with your encryption key to the `wrapper.conf` file of your Mule environment.

> When the API gateway capability is starting, it checks for an encryption key. The client ID and client secret values are then obtained from its `wrapper.conf` file. If some of the values in that file are enclosed within the encryption indicators (`![` and `]`), the Mule environment uses the encryption key that it loaded earlier to decrypt the values and use its original content.

Reviewer 2: ```suggestion
When the API gateway capability starts, it checks for an encryption key. It then obtains the client ID and client secret values from its `wrapper.conf` file. If any of the values in that file are enclosed within encryption indicators (`![` and `]`), Mule uses the encryption key that it loaded earlier to decrypt the values and use its original content.
```

Author: Done! Thanks!

Client ID cannot be decrypted
----

> If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform. Subsequently you will not be able to download policies or contracts, or generate analytics information for tracked APIs.

Reviewer 2: ```suggestion
If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform, which means that you cannot download policies or contracts or generate analytics information for tracked APIs.
```

Author: Kept as is because I feel when the sentence is too long, comprehension is impaired.

> If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform. Subsequently you will not be able to download policies or contracts, or generate analytics information for tracked APIs.

Reviewer 2: ```suggestion
If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform. Subsequently, you cannot download policies or contracts, or generate analytics information for tracked APIs.
```

Reviewer 2: I liked my previous rewording better, however....
```suggestion
If your credentials cannot be decrypted, you cannot establish a connection with Anypoint Platform, nor can you subsequently download policies or contracts or generate analytics information for tracked APIs.
```

Author: Done! Thanks!

[%header%autowidth.spread,cols="a,a,a"]
|===

> | Property | Property Key | Description

Reviewer 2: Table column heads use title-style capitalization, but table Row 1 entries use sentence-style capitalization.

Author: Done! Thanks!

| Client ID | `anypoint.platform.client_id` | Your Anypoint Platform Client ID

> | Client Secret | `anypoint.platform.client_secret` | Your Anypoint Platform Client Secret

Reviewer 2: ```suggestion
| Client secret | `anypoint.platform.client_secret` | Your Anypoint Platform client secret
```

Author: Done! Thanks!

> | Proxy Password | `anypoint.platform.proxy_password` | Password of the proxy configuration to connect to Anypoint Platform

Reviewer 2: ```suggestion
| Proxy password | `anypoint.platform.proxy_password` | Password of the proxy configuration to connect to Anypoint Platform
```

Author: Done! Thanks!

To enable encryption in policies, you need to provide two system properties in your `wrapper.conf` file:

* `anypoint.platform.encryption_key.<n>` +

> The encryption key for your policy.

Reviewer: Same comment about the use of <n> as before.

Author: Done! Thanks!

This is the default encryption mode.
|===
+

> If this property is not set, the default `SENSITIVE_ONLY` encryption mode is used. +

Reviewer 2: Consider rewording to use active voice.

Author: Done! Thanks!
